samples/queries/input/SelfJoin.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
```

Comment:
Can we please use a simpler example here. (See readme comment)

Reply:
used simple examples